File: src/net/eduard/api/server/RankSystem.java
package net.eduard.api.server;

import net.eduard.api.lib.modules.FakePlayer;

import java.util.List;

public interface RankSystem {

    public String getRank(FakePlayer player);

    public String getRankPrefix(FakePlayer player);

    public String getRankSuffix(FakePlayer player);

    public double getRankProgress(FakePlayer player);

    public List<String> getRanksNames();


}

File: src/net/eduard/api/server/RankSystem.kt
package net.eduard.api.server

import net.eduard.api.lib.player.FakePlayer

interface RankSystem {

    val ranksNames: List<String>

    fun getRank(player: FakePlayer): String

    fun getRankPrefix(player: FakePlayer): String

    fun getRankSuffix(player: FakePlayer): String

    fun getRankProgress(player: FakePlayer): Double


}
